api/relations/v0/lookup.proto

```diff
@@ -17,6 +17,24 @@ service KesselLookupService {
             get: "/v0/subjects"
         };
     };
+    rpc LookupResources(LookupResourcesRequest) returns (stream LookupResourcesResponse) {
+        option (google.api.http) = {
+            get: "/v0/resources"
+        };
+    };
+}
+
+message LookupResourcesRequest{
+    ObjectReference resource = 1 [(validate.rules).any.required = true];
+    string relation = 2 [(validate.rules).string.min_len = 1];
+    ObjectType subject_type = 3 [(validate.rules).any.required = true];
+    optional string subject_relation = 4;
```

Comment:
Lookup resources needs to be kind of the "reverse" of lookup subjects. So, instead of specifying a specific ObjectReference, you need just an ObjectType for the resource. And for the subject, instead of only the type (and optional relation), you need a SubjectReference for a specific subject.